lib/mutant/subject/method/instance.rb

```diff
@@ -42,7 +42,7 @@ def prepare
         private
 
           def wrap_node(mutant)
-            s(:begin, mutant, s(:send, nil, :memoize, s(:args, s(:sym, name), *options)))
```

Comment:
@snusnu All this years we where generating *invalid AST* that would silently be emitted as correct intended source.